docs/source/features/version_control/draft.rst

```diff
@@ -0,0 +1,121 @@
+:class:`~graviti.manager.draft.Draft` can be viewed as a writable dataset in SDK.
+
+The following sections will introduce the operations and precautions related to drafts
+in the SDK. Of course, it is necessary to get a dataset first:
+
+.. code:: python
+
+   from graviti import Workspace
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.get(f"{DATASET_NAME}")
+
+*****************
+ Create a Draft
+*****************
+
+SDK provides method :meth:`~graviti.manager.draft.DraftManager.create` to support creating a
+draft based on a branch. The title, description and branch of the draft can be specified, but
+only title is required:
+
+.. code:: python
+
+   dataset.drafts.create(f"{DRAFT_TITLE}")
+   dataset.drafts.create(f"{DRAFT_TITLE}", f"{DRAFT_DESCRIPTION}")
+   dataset.drafts.create(f"{DRAFT_TITLE}", branch=f"{BRANCH_NAME}")
+
+.. note::
+   If no branch is specified, the created draft will be based on the current branch of the
+   dataset, which can be viewed by ``dataset.HEAD``.
+
+.. important::
+   Graviti use number, not title, to uniquely identify drafts.
+
+*************
+ List Drafts
+*************
+
+SDK provides method :meth:`~graviti.manager.draft.DraftManager.list` to support listing drafts.
+Drafts can be filtered by state (``OPEN``, ``COMMITTED``, ``CLOSED`` or ``ALL``) and branch.
+The default is to list open drafts on all branches.
+
+.. code:: python
+
+   dataset.drafts.list()
+   dataset.drafts.list(f"{STATE}")
+   dataset.drafts.list(branch=f"{BRANCH_NAME}")
+
+*************
+ Get a Draft
+*************
+
+SDK provides method :meth:`~graviti.manager.draft.DraftManager.get` to support getting a draft:
+
+.. code:: python
+
+   dataset.drafts.get(DRAFT_NUMBER)
+
+****************
+ Edit the Draft
+****************
+
+SDK provides method :meth:`~graviti.manager.draft.Draft.edit` to support changing the title and
+description of the draft:
+
+.. code:: python
+
+   draft = dataset.drafts.get(DRAFT_NUMBER)
+   draft.edit(f"{NEW_TITLE}", f"{NEW_DESCRIPTION}")
+
+******************
+ Upload the Draft
+******************
+
+SDK provides method :meth:`~graviti.manager.draft.Draft.upload` to support uploading the local
+draft to Graviti. This step is essential if the user wants to save changes to the sheet
+of the dataset.
+
+.. code:: python
+
+   draft = dataset.drafts.get(DRAFT_NUMBER)
+   del draft["train"]
+   draft.upload()
+
+.. note::
+   SDK supports specifying the max workers in multi-thread upload. The default is 8.
+
+******************
+ Commit the Draft
+******************
+
+SDK provides method :meth:`~graviti.manager.draft.Draft.commit` to support committing a draft:
+
+.. code:: python
+
+   draft = dataset.drafts.get(DRAFT_NUMBER)
+   draft.commit(f"{COMMIT_TITLE}")
+   draft.commit(f"{COMMIT_TITLE}", f"{COMMIT_DESCRIPTION}")
+
+.. warning::
+   It is not allowed to read or upload data on a committed draft.
+
+*****************
+ Close the Draft
```

Comment:
Do we need to demonstrate the difference between closing a draft and committing a draft?